pkg/iac/scanners/dockerfile/parser/parser_test.go

```diff
@@ -59,3 +59,60 @@ CMD python /app/app.py
 	assert.Equal(t, 4, commands[3].StartLine)
 	assert.Equal(t, 4, commands[3].EndLine)
 }
+
+func Test_ParseHeredocs(t *testing.T) {
+	tests := []struct {
+		name     string
+		src      string
+		expected string
+	}{
+		{
+			name: "multi-line script",
+			src: `RUN <<EOF
+apk add curl
+apk add git
+EOF`,
+			expected: "apk add curl ; apk add git",
+		},
+		{
+			name: "file redirection and chained command",
+			src: `RUN cat <<EOF > /tmp/output && echo 'done'
+hello
+mr
+potato
+EOF`,
+			expected: "cat <<EOF > /tmp/output && echo 'done'\nhello\nmr\npotato\nEOF",
+		},
+		{
+			name: "redirect to file",
+			src: `RUN <<EOF > /etc/config.yaml
+key1: value1
+key2: value2
+EOF`,
+			expected: "<<EOF > /etc/config.yaml\nkey1: value1\nkey2: value2\nEOF",
+		},
+		{
+			name: "with a shebang",
+			src: `RUN <<EOF
+#!/usr/bin/env python
+print("hello world")
+EOF`,
+			expected: "/dev/pipes/EOF",
```

Comment:
I read your description for the case when it starts with a shebang, but I don't understand this test case. Why is the expected value the one it is?